tests/sentry/middleware/test_ratelimit_middleware.py

```diff
@@ -239,9 +240,9 @@ def test_get_rate_limit_key(self):
         )
 
         self.populate_internal_integration_request(request)
-        assert (
+        key_pattern = re.compile(r"^org:default:OrganizationGroupIndexEndpoint:GET:[a-zA-Z]$")
```

Comment:
Match the temporary fix of avoiding rate limit key collisions by using regex that was done in https://github.com/getsentry/sentry/pull/53673/files